src/packages/@ncigdc/dux/bannerNotification.js

```diff
@@ -33,7 +33,7 @@ let initialState = [
   {
     components: ['PORTAL'],
     level: 'INFO',
-    id: 0,
+    id: 'initial_banner',
     dismissible: true,
     message: <DismissibleBanner />,
```

Comment:
I don't think we do.. but I like more verbose initial state if that's an important thing